docs/source/deployment/amazon_emr_serverless.md

```diff
@@ -0,0 +1,242 @@
+        }
+    }'
+```
+
+Enter the respective values in the placeholders above, e.g. use the ARN from the job runtime role created earlier for
+`<execution-role-arn>`.
+
+## FAQ
+
+### How is the approach defined here different from [this approach](https://kedro.org/blog/how-to-deploy-kedro-pipelines-on-amazon-emr) on the Kedro blog?
+
+There are similarities in steps in both approaches. However, the key difference is that this page explains how to provide a custom Python version
+through the custom image. The blog post provides Python dependencies in a virtual environment for EMR to refer to.
+
+The approach of providing a custom image is only applicable to EMR Serverless. On EMR it would be worth considering [using a custom AMI](https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-custom-ami.html), as an alternative to using [bootstrap actions](https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-plan-bootstrap.html). However, this has not been explored nor tested as yet.
+
+### EMR Serverless already has Python installed. Why do we need a custom Python version?
+
+The code base in the example requires Python 3.9 while the default Python installation on the latest EMR releases (6.10.0, 6.11.0) is based on Python 3.7.
+
+### Why do we need to create a custom image to provide the custom Python version?
+
+You may encounter difficulties with the [virtual environment approach](https://docs.aws.amazon.com/emr/latest/EMR-Serverless-UserGuide/using-python.html), particularly when modifying it to use [pyenv](https://github.com/pyenv/pyenv). While using [venv-pack](https://pypi.org/project/venv-pack/) we found [a limitation that returned the error `Too many levels of symbolic links`](https://jcristharif.com/venv-pack/#caveats) which is documented as follows:
+
+>Python is _not_ packaged with the environment, but rather symlinked in the environment.
+> This is useful for deployment situations where Python is already installed on the machine, but the required library dependencies may not be.
+
+
+The custom image approach neatly includes both installing the Kedro project and provides
+the custom Python version in one go. Otherwise, you would need to separately provide the files, and specify them as extra Spark configuration each time you submit a job.
+
+### Why do we need to package the Kedro project and invoke using an entrypoint script? Why can't we just use [CMD] or [ENTRYPOINT] with `kedro run` in the custom image?
+
+As mentioned in the [AWS documentation](https://docs.aws.amazon.com/emr/latest/EMR-Serverless-UserGuide/application-custom-image.html), EMR Serverless ignores `[CMD]` or `[ENTRYPOINT]` instructions in the Dockerfile.
+
+You should prefer using the `entrypoint.py` entrypoint script approach to run Kedro
```

Comment:
```suggestion
It is also recommended to use the `entrypoint.py` entrypoint script approach to run Kedro
```

Reply:
Just wanted to avoid being too prescriptive 🙂
Or maybe we can make it passive as well: "It is preferred to use..."